src/modules/endpoint/DatabaseConnector.js

```diff
@@ -339,6 +352,46 @@ class DatabaseConnector {
     const path = this.getContentPath('languages', context)
     await this.writeFile(path, JSON.stringify(languages))
   }
+  
+  async storePois (pois: Array<PoiModel>, context: DatabaseContext) {
+    const jsonModels = pois.map((poi: PoiModel): ContentPoiJsonType => ({
+      path: poi.path,
+      title: poi.title,
+      content: poi.content,
+      thumbnail: poi.thumbnail,
+      availableLanguages: poi.availableLanguages,
+      excerpt: poi.excerpt,
+      location: poi.location,
+      lastUpdate: poi.lastUpdate,
```

Comment:
What is the format of this timestamp? You probably need to store it in ISO8601: `object.lastUpdate.toISOString()`